src/modules/simple-lock/services/simple.lock.abi.service.ts

```diff
@@ -39,6 +44,18 @@ export class SimpleLockAbiService
         return response.firstValue.valueOf().toString();
     }
 
+    async getAllLockedTokenIds(
```

Comment:
I don't think it's worth having this type of methods in here. We only have 3 simple lock contracts

Reply:
removed all 3 methods